### PR TITLE
Uuid Validator

### PR DESCRIPTION
**Pull request by gsomoza**
I realise there's an open road to ZF3 PR but here's what I'd consider some low-hanging fruit in case you're planning any other 2.x releases. 

NOTE: I'd be happy to upgrade this to the new format once the new API for validators is stabilised in that other PR. So if you don't think you'll release this in a new 2.x release let me know and I'll rebase the work against the ZF3 WIP branch.

### CODE REVIEW

File: test/UuidTest.php [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+    }
+
+    /**
+     * providesValidUuids.
+     *
+     * @return array
+     */
+    public function validUuidProvider()
+    {
+        return array(
```
Comment (samsonasik) [inline test/UuidTest.php]: can use short array syntax `[]`

Comment (gsomoza) [inline test/UuidTest.php]: Done!

**Comment by weierophinney** [comment]
Thanks, @gsomoza !

**Comment by gsomoza** [comment]
It's nothing! Happy to help a little bit :)

**Pull request merged**